docs/flutter_status.md

```diff
@@ -0,0 +1,98 @@
+# Current Flutter status
+
+## Firebase
+
+1. Firebase can run flutter tests. You can find example in ```test_projects/flutter```,
+   simple run ```./build_and_run_tests_firebase```.
+
+1. Firebase not supporting sharding for Flutter.
```

Comment:
This could be explained in more detail.
Add script with commands used for tests, also the output with error will be useful.
Or direct link to firebase documentation explaining that is not supported.

Reply:
Thanks! I added a command example and describe test lab behavior.